### PR TITLE
One Service

### PR DESCRIPTION
**Pull request by oliverlevay**
This combines our backend services to one service.

`gateway`, `migrator`, `calendar`, `news`, `booking` and `file-handler` all become a part of `core`

The purpose is to make our builds faster on jenkins and reduce complexity, especially when starting out. Hopefully this will help new people join the project.

We're waiting for @01ste02 to make the required changes in jenkins before we can merge.

### CODE REVIEW

**Comment by 01ste02** [comment]
Jenkins is ready, and after merge we simply need to wait for a test-build to complete before changing the deploy-pipeline and actually pushing to production

**Review by LucasBoberg — CHANGES_REQUESTED** [review]
I have one comment, that I would like to see changed. And I would like for one more knowledgeable in docker to verify the docker files

**Review by 01ste02 — COMMENTED** [review]
The dockerfiles look good to me, and are building properly. There remains one issue with the frontend-container not getting env-values from the env-file, but rather relying on a static env-file that is copied on build instead of one that is mounted at runtime. However, that is not related to the dockerfiles.

TLDR; Dockerfiles seem to be OK, and are building

File: backend/services/core/src/migrator/searchData.ts [hunk 1]
```diff
@@ -1,8 +1,6 @@
-async function meilisearchSeed(knex, meilisearch) {
+export default async function meilisearchSeed(knex: any, meilisearch: any) {
```
[the remark this answers is not in the document]
Comment (oliverlevay) [reply backend/services/core/src/migrator/searchData.ts:1]: > I have one comment, that I would like to see changed. And I would like for one more knowledgeable in docker to verify the docker files

You can take a look, @01ste02 created a sandbox environment based on this branch and everything seems to be working https://member.sandbox.dsek.se/ , I'll fix the types.

**Comment by oliverlevay** [comment]
> I have one comment, that I would like to see changed. And I would like for one more knowledgeable in docker to verify the docker files

You can take a look, @01ste02 created a sandbox environment based on this branch and everything seems to be working https://member.sandbox.dsek.se/
